### PR TITLE
add static component when the page loads

### PR DESCRIPTION
**Pull request by qlj-lijuan**
- add the static component when the page loads, replace the old url with the new ones.
- in the url, join the node path by " ' " instead of " , " , because " , " is used in the income range such as "Less than 9,999 $"

### CODE REVIEW

File: static/js/tools/timeline_page.tsx [hunk 1]
```diff
@@ -125,6 +126,7 @@ class Page extends Component<PagePropType, PageStateType> {
             <div id="search">
               <SearchBar places={this.state.places} />
             </div>
+            {(window.location.hash === "") && <Info/>}
```
Comment (shifucun) [inline static/js/tools/timeline_page.tsx:129]: check this.state.places.length === 0? There is the case when stats var is selected but place are removed

File: static/js/tools/statsvar_menu.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Node extends Component<NodePropType, NodeStateType> {
     this.state = {
       checked: false,
       expanded: false,
-      nodePath: props.nodePath + "," + props.title,
+      nodePath: props.nodePath + "'" + props.title,
```
Comment (shifucun) [inline static/js/tools/statsvar_menu.tsx:31]: can you make this a constant? like const SEP = "'". May need to change it in the future if we have ' in the string..

File: static/js/tools/timeline_page.tsx (hunk 1 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_page.tsx]: done.

File: static/js/tools/statsvar_menu.tsx (hunk 2 above)
Comment (qlj-lijuan) [reply static/js/tools/statsvar_menu.tsx]: done.

File: static/js/tools/timeline_util.test.ts [hunk 3]
```diff
@@ -22,20 +23,24 @@ test("update Url statsvar", () => {
 });
 
 test("parse statvar from Url", () => {
-  window.location.hash = "#&statsvar=dc/test,Demo,prop";
+  window.location.hash = "#&statsvar=dc/test'Demo'prop";
```
Comment (shifucun) [inline static/js/tools/timeline_util.test.ts:26]: also need to change here

File: static/js/tools/timeline_info.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                    find data about places (zip codes, cities, counties, states) from a variety of
+                    sources, including the Census, FBI, Bureau of Labor Statistics, CDC and others. </p>
+                <p> Enter a place in the search box above and then pick one or more of the variables in
+                    the pane to the left. There are thousands of variables to choose from, arranged in a
+                    topical hierarchy. </p>
+                <p> Or you can start your exploration from these interesting points ...</p>
+                <ul>
+                    <li>
+                        <b>University towns</b> by
+                        <a href="#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsvar=MedianAge'Demographics'Median%20age"> age</a>,
```
Comment (shifucun) [inline static/js/tools/timeline_info.tsx:18]: please also change here. As this is used a lot, can think of a short name for it (2 or 3 letters)

File: static/js/tools/timeline_util.test.ts (hunk 3 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_util.test.ts]: done

File: static/js/tools/timeline_info.tsx (hunk 4 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_info.tsx]: done

**Pull request merged**